libs/utils/utils.js

```diff
@@ -940,6 +939,9 @@ async function documentPostSectionLoading(config) {
     const { default: loadGeoRouting } = await import('../features/georoutingv2/georoutingv2.js');
     loadGeoRouting(config, createTag, getMetadata, loadBlock, loadStyle);
   }
+
+  await decorateFooterPromo(config);
```

Comment:
Since you now load fragment.js directly, I think we can remove the `await`?   It's preferable for the code below to keep running while footer-promo.js loads.

Reply:
Good catch! Updated